tflint/README.md

```diff
@@ -0,0 +1,49 @@
+# tflint 
+
+This package is a wrapper of the ![tflint](https://github.com/terraform-linters/tflint) project, enabling it to be natively executed from the before and after hooks. The `tflint/tflint.go` uses the MPL 2.0 license, due to the license of tflint.
+
+## How to use
```

Comment:
These docs don't belong here. They belong in the docs site. See the `docs` folder.